### PR TITLE
Updated ssh.rst to include instruction on finding IP address of UT de…

### PR DESCRIPTION
**Pull request by sanathusk**
…vice

Updated ssh.rst to include instruction on finding IP address of UT device when needed to connect through ssh.

### CODE REVIEW

File: userguide/advanceduse/ssh.rst [hunk 1]
```diff
@@ -22,6 +22,9 @@ You need then to transfer your public key to your device. There are multiple way
 * You can also connect via :doc:`adb <adb>` and use the following command to copy it::
 
     adb push ~/.ssh/id_rsa.pub /home/phablet/
+    
+Please note:Some of the UT devices might not deteched on connecting through USB or adb devices might not list your device,In such situations transfering the key via the internet to your phone is a better approcah.
+
```
Comment (doniks) [inline userguide/advanceduse/ssh.rst:27]: Doesn't the sentence above "Or transfer ..." clarify this already?

**Review by doniks — CHANGES_REQUESTED** [review]

File: userguide/advanceduse/ssh.rst (hunk 1 above)
Comment (sanathusk) [reply userguide/advanceduse/ssh.rst:27]: I had seen a couple of queries in the telegram group around not being able to connect through adb. Users could be of the impression that adb would work even in UT as they would have connected to the same phone through adb in android.
Mentioning it explicitly in dcoumentation would perhaps reduce the confusion.

Comment (doniks) [reply userguide/advanceduse/ssh.rst:27]: Ok. I can imagine there is *some* way to make it more explicit, but in the proposed form it looks really strange. 

> Do this thing. There are a few options:
> * do A 
> * do B 
> * do C

and now we add a sentence at the bottom:

> Oh and btw if you have trouble with option C, why not try option A.

Comment (sanathusk) [reply userguide/advanceduse/ssh.rst]: I removed the statement.Please review the rest.

File: userguide/advanceduse/ssh.rst [hunk 2]
```diff
@@ -50,6 +50,13 @@ Connect
 Now everything is set up and you can use ``ssh`` ::
 
     ssh phablet@<ip-address>
+    
+To identify the ip-address of your UT devce,open Termux application in your phone and run the following command::
```
Comment (doniks) [inline userguide/advanceduse/ssh.rst:54]: (I could have sworn I already posted this ... but it is nowhere to be seen ... oh well, here it goes again)

devce -> device
space after the comma
I think Termux is some android app. I guess you mean Terminal

File: userguide/advanceduse/ssh.rst [hunk 3]
```diff
@@ -50,6 +50,13 @@ Connect
...
+    
+    hostname -I
+    
+The output is list on IP address seperated by spaces.Use the first IP address from the list.
```
Comment (doniks) [inline userguide/advanceduse/ssh.rst:58]: is **a** list **of** IP address**es**
space after the dot

File: userguide/advanceduse/ssh.rst [hunk 4]
```diff
@@ -15,7 +15,7 @@ If not already created, create your public key, default choices should be fine f
 Copy the public key to your device
 ----------------------------------
 
-You need then to transfer your public key to your device. There are multiple ways to do this. For example:
+You need then to transfer your public key to your device. There are multiple ways to do this. These are the few options:
```
Comment (doniks) [inline userguide/advanceduse/ssh.rst:18]: the few -> a few

**Review by doniks — CHANGES_REQUESTED** [review]

File: userguide/advanceduse/ssh.rst (hunk 4 above)
Comment (sanathusk) [reply userguide/advanceduse/ssh.rst]: made the suggested changes.

File: userguide/advanceduse/ssh.rst (hunk 2 above)
Comment (doniks) [reply userguide/advanceduse/ssh.rst:54]: what about this?

File: userguide/advanceduse/ssh.rst (hunk 3 above)
Comment (doniks) [reply userguide/advanceduse/ssh.rst]: what about this?

**Comment by sanathusk** [comment]
I have made the requested changes.Please review the same.

**Comment by sanathusk** [comment]
Made the following  recommended changes.
" a few typos in ssh.rst 
on line 54 'of your UT devce,open terminal' should be 'of your UT device, open Terminal'(note the space after the comma aswell)
on line 58 
'a list on IP adresses' should be "a list of IP adresses"

File: userguide/advanceduse/ssh.rst [hunk 5]
```diff
@@ -50,6 +50,13 @@ Connect
 Now everything is set up and you can use ``ssh`` ::
 
     ssh phablet@<ip-address>
+    
+To identify the ip-address of your UT device , open Terminal application in your phone and run the following command::
+    
+    hostname -I
+    
+The output is a list of IP addresses seperated by spaces.Use the first IP address from the list.
```
Comment (doniks) [inline userguide/advanceduse/ssh.rst:58]: I only notice now, sorry - it's "separated" - second vowel is an a not e. 

Space before "Use" please.

Comment (sanathusk) [reply userguide/advanceduse/ssh.rst]: Made typo correction & addeded the space before use

**Review by doniks — APPROVED** [review]
Thx!

**Pull request merged**